packages/shared/lib/services/notification/slack.service.ts

```diff
@@ -114,53 +121,40 @@ export class SlackService {
      *      so future notifications can be sent as updates to the original.
      *      3) Add an activity log entry for the notification to the admin account
      */
-    async reportFailure(nangoConnection: ConnectionJobs, name: string, type: string, originalActivityLogId: string, provider: string) {
-        // WARNING: disabling slack notifications while investigating sync issues
-        if (this.disabled) {
-            return;
-        }
-
-        if (name === this.actionName) {
+    async reportFailure({
+        account,
+        environment,
+        provider,
+        name,
+        connection,
+        type,
+        originalActivityLogId
+    }: {
+        account: DBTeam;
+        environment: DBEnvironment;
+        connection: ConnectionJobs;
+        name: string;
+        type: string;
+        originalActivityLogId: string;
+        provider: string;
+    }) {
+        if (await this.isDisabled()) {
+            console.log('>>>>>>>>>>>>>>>>>>>Slack notifications are disabled');
```

Comment:
not sure if >>> is intentional :D
maybe it could be more useful for debugging to add more info, e.g:
```suggestion
            logger.info('Disabled, not sending failure for ${name} ${type}');
```

Reply:
print debugging!!!!!!!!!!!! 🙈